moped-editor/src/views/projects/projectView/DateFieldEditComponent.js

```diff
@@ -1,6 +1,5 @@
 import React from "react";
-import DateFnsUtils from "@date-io/date-fns";
-import { DatePicker, MuiPickersUtilsProvider } from "@material-ui/pickers";
+import { MobileDatePicker } from "@mui/x-date-pickers";
```

Comment:
chose this as the date picker to match the one we have in prod now

Reply:
It looks really good, thank you!

Reply:
@chiaberry apparently the `clearable` and `emptyLable` do nothing. to make the mobile date picker clearable, you have to do this:

```jsx
    <MobileDatePicker
      clearable={true}
      format="MM/dd/yyyy"
      value={props.value ? parseISO(props.value) : null}
      onChange={handleDateChange}
      InputProps={{ style: { minWidth: "100px" } }}
      slotProps={{
        actionBar: {
          actions: ["accept", "cancel", "clear"],
        },
      }}
    />
````

 😵

Reply:
thank you for this! updated it.